views/partials/player_card.hbs

```diff
@@ -36,6 +37,7 @@
             <td id="fg-percent">\{{percentageCalc average.field_goals_att average.field_goals_made}}</td>
             <td id="ft-percent">\{{percentageCalc average.free_throws_att average.free_throws_made}}</td>
             <td id="threepoint-percent">\{{percentageCalc average.three_points_att average.three_points_made}}</td>
+            <td id="salary">\{{salary}}</td>
```

Comment:
It's part of the players object. fE gets it the same way it gets these free throw % and such 

Kelsey 
Edmodo Customer Success

> On Apr 27, 2015, at 2:29 PM, Blake Parkinson <notifications@github.com> wrote:
> 
> In views/partials/player_card.hbs:
> 
> > @@ -36,6 +37,7 @@
> >              <td id="fg-percent">\{{percentageCalc average.field_goals_att average.field_goals_made}}</td>
> >              <td id="ft-percent">\{{percentageCalc average.free_throws_att average.free_throws_made}}</td>
> >              <td id="threepoint-percent">\{{percentageCalc average.three_points_att average.three_points_made}}</td>
> > +            <td id="salary">\{{salary}}</td>
> i dont see this being passed to the FE anywhere?
> 
> —
> Reply to this email directly or view it on GitHub.
>